cicd/internal/op/maven.go

```diff
@@ -29,18 +31,18 @@ func RunMavenOnPom(pom string, cmd string, args ...string) error {
 	fullArgs = append(fullArgs, "-f", pom)
 	fullArgs = append(fullArgs, "-e")
 	fullArgs = append(fullArgs, args...)
-
+	modules := flags.ModulesToBuild()
+	if len(modules) != 0 {
+		moduleArgs := []string{"-pl", strings.Join(modules, ",")}
+		fullArgs = append(fullArgs, moduleArgs...)
+		fullArgs = append(fullArgs, "-am")
```

Comment:
Oh nice - this is cleaner (both `-am` and moving it into `RunMavenOnPom`). Thanks!

Reply:
Thanks!